example/windows/win32_window.cc

```diff
@@ -4,23 +4,41 @@
 
 #include "win32_window.h"
 
+#include <flutter_windows.h>
+
 #include "resource.h"
-#include "shellscalingapi.h"
```

Comment:
Presumably you also need to make a project change to stop linking directly to Shcore.lib?